equinix/resource_metal_device_acc_test.go

```diff
@@ -1210,3 +1168,60 @@ func TestAccMetalDeviceUpdate_timeout(t *testing.T) {
 		},
 	})
 }
+
+func TestAccMetalDevice_LockingAndUnlocking(t *testing.T) {
```

Comment:
lock/unlock are properties of a device. It would be sufficient to bake these test steps (and similar non-destructive attribute modifications) into an existing TestAccMetalDevice test.  I realize that defeats some of the benefits of testing in isolation, but it also reduces the amount of physical servers we need to occupy for real time during tests. 

It's a balancing act. Perhaps we can address that in a separate E2E improvement PR.

Reply:
In this case, we are verifying that `destroy` is blocked by the lock. I overlooked that. This test is very reasonable.